qemu/tests/cfg/balloon_service.cfg

```diff
@@ -0,0 +1,22 @@
+- balloon_service: install setup image_copy unattended_install.cdrom
+    virt_test_type = qemu
+    type = balloon_service
+    only Windows
+    driver_name = "balloon"
+    balloon = balloon0
+    balloon_dev_devid = balloon0
+    balloon_dev_add_bus = yes
+    need_enable_verifier = yes
+    need_clear_verifier = yes
+    cdroms = "cd1 winutils virtiowin"
+    drive_index_virtiowin = 3
+    cdrom_virtiowin = iso/windows/virtio-win-1.8.0-4.iso
+    # balloon service related cmd
+    install_balloon_service = "E:\Balloon\w7\amd64\blnsvr.exe -i"
```

Comment:
Better to get driver letter by  python code.  We need to make our code work in PPC and ARM platform.